src/components/MainComponents/Button.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components'
```

Comment:
It button.js a leftover  from team session? Because I can not find it in the project? Could in that case be omitted because it's not used?

Reply:
Yes.... didn't do my clean up....